assets/data/game.ron

```diff
@@ -253,7 +254,7 @@ GameData(
         "crab",
         "criminal",
         "crooked",
-        "cunning",
```

Comment:
slightly over-zealous search/replace

Reply:
hehe, yep. I noticed `Annotated` -> `Anotated` but forgot to check the english words.